main.go

```diff
@@ -78,6 +83,10 @@ func Main() int {
 		flag.Parse()
 	}
 
+	if exitWithParent {
+		system.ParentDeathSignal(unix.SIGTERM).Set()
```

Comment:
one line not covered by tests. :P

Reply:
Actually, it's perhaps worth testing it. Start slirpnetstack with some dummy options under bash (os.system(bash -C "slirpnetstack --")) (or something else?) kill bash. Validate if the slirp died. 

Or maybe not... because it's such a simple funnctionality....

Reply:
I will try to test it, always interesting exercise anyway

Reply:
I would expect this to work, but it doesn't on my machine.

```py
    def test_exit_with_parent(self):
        ''' Test whether process exits with parent. '''
        c1, c2 = socket.socketpair(socket.AF_UNIX)
        os.set_inheritable(c2.fileno(), True)
        pid = os.fork()
        if pid == 0:
            p = self.prun("-exit-with-parent", close_fds=False)
            self.assertStartSync(p)
            c2.send(p.p.pid.to_bytes(8, byteorder='little'))
            c2.recv(1) # wait
        else:
            cpid = int.from_bytes(c1.recv(8), byteorder='little')
            os.kill(pid, signal.SIGTERM)
            os.waitpid(pid, 0)
            with self.assertRaises(OSError):
                os.kill(cpid, 0)
        c1.close()
        c2.close()
```

It seems the grand-child is reaped by systemd -user, and doesn't receive SIGTERM.

yet, sometime it does work, when running more interactively (with input/debug/timers)...

iow, -exit-with-parent works only under some circunstances that aren't clear to me :)